xpholder/commands/mod/viewGameRules.js

```diff
@@ -1,243 +1,401 @@
-            { inline: true, name: "Mod Role", value: `<@&${configObj.moderationRoleId}>` },
-            { inline: true, name: "Role Bonus", value: `${configObj.roleBonus}` },
-            { inline: true, name: "Tier 1", value: `<@&${configObj.tier1RoleId}>` },
-            { inline: true, name: "Tier 2", value: `<@&${configObj.tier2RoleId}>` },
-            { inline: true, name: "Tier 3", value: `<@&${configObj.tier3RoleId}>` },
-            { inline: true, name: "Tier 4", value: `<@&${configObj.tier4RoleId}>` },
-            { inline: true, name: "Allow Player Manage Xp", value: `${configObj.allowPlayerManageXp}` },
-            { inline: true, name: "XP Freeze", value: `<@&${configObj.xpFreezeRoleId}>` },
-            { inline: true, name: "XP Share", value: `<@&${configObj.xpShareRoleId}>` },
-            { inline: true, name: "XP Per-Post Divisor", value: `${configObj.xpPerPostDivisor}` },
-            { inline: true, name: "XP Per-Post Formula", value: `${configObj.xpPerPostFormula}` },
-        )
-        .setFooter({ text: `Dont Like What You See? Try : /edit_config` })
-        .setThumbnail(XPHOLDER_ICON_URL)
-        .setColor(XPHOLDER_COLOUR)
-        .setURL(DEV_SERVER_URL);
+${configObj.approveMessage}`
+    )
+    .setFields(
+      {
+        inline: true,
+        name: "Approve Level",
+        value: `${configObj.approveLevel}`,
+      },
+      {
+        inline: true,
+        name: "Character 1",
+        value: `<@&${configObj.character1RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 2",
+        value: `<@&${configObj.character2RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 3",
+        value: `<@&${configObj.character3RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 4",
+        value: `<@&${configObj.character4RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 5",
+        value: `<@&${configObj.character5RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 6",
+        value: `<@&${configObj.character6RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 7",
+        value: `<@&${configObj.character7RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 8",
+        value: `<@&${configObj.character8RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 9",
+        value: `<@&${configObj.character9RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Character 10",
+        value: `<@&${configObj.character10RoleId}>`,
+      },
+      {
+        inline: true,
+        name: "Level-Up Channel",
+        value: `<#${configObj.levelUpChannelId}>`,
+      },
+      {
+        inline: true,
+        name: "Mod Role",
+        value: `<@&${configObj.moderationRoleId}>`,
+      },
+      { inline: true, name: "Role Bonus", value: `${configObj.roleBonus}` },
```

Comment:
Short answer, yes they are redundant. This feature attempts to move them to a more flexible way of handling tier roles. In this release, both versions still exist, as I'm not 100% certain of what else the old version of the roles affects and/or what it would take to extract them and maintain all current functionality. At a later point, my intention is to remove the old version.